lib/nexmo_markdown_renderer/config/load_config.rb

```diff
@@ -1,8 +1,17 @@
 module Nexmo
   module Markdown
     class Config
+      def self.mounted?
+        defined?(Rails) && !Rails.application.nil?
+      end
+    
       def self.docs_base_path
-        @docs_base_path ||= (defined?(Rails) && Rails.application != nil && Rails.application.configuration.docs_base_path) || ENV.fetch('DOCS_BASE_PATH', '.')
+        (mounted? && Rails.application.configuration.docs_base_path) || '.'
```

Comment:
@benhayehudi `Rails.application.configuration` is raising an exception, it should be `Rails.application.config`